### ISSUE TITLE
Update to recent HTML_CodeSniffer and use it as a dependency

### ISSUE THREAD

**Issue by paazmaya** [pa11y/pa11y#448]
## Expected behaviour

Have a recent version of `HTML_CodeSniffer` used in this project.

## Actual behaviour

Using a slightly older version, which is manually copied and comitted in this project code.

## Steps to reproduce

Looking to `lib` folder and in `package.json`...

## Environment:

```
Pa11y:      5.1.0
Node.js:    8.15.1
npm:        6.9.0
OS:         18.5.0 (darwin)
```

**Comment by paazmaya** [comment]
Shall I create PR for this?

**Comment by hollsk** [comment]
Yes please, but I think there's a reason we do it in the weird way that we do (we _used_ to have it as a dependency). I don't remember what the reason is, though. 

@rowanmanning what was the reason for not using HTML_CS as a normal dep?

**Comment by paazmaya** [comment]
Perhaps because it (`HTML_CodeSniffer`) did not provide build version of itself in its npm package...

https://github.com/squizlabs/HTML_CodeSniffer/releases/tag/2.3.0

**Comment by hollsk** [comment]
Haha! Yes! That _is_ why - I was thinking to myself "... it's something to do with the build, isn't it?" 

But yes, that's the reason. Good catch!

**Comment by paazmaya** [comment]
CLosing this since the PR #460 is merged.
Might fix #420 too...
Would it make sense to release a new version now?
